workflows/sarek.nf

```diff
@@ -52,7 +52,7 @@ for (param in checkPathParamList) if (param) file(param, checkIfExists: true)
 // Set input, can either be from --input or from automatic retrieval in WorkflowSarek.groovy
 ch_input_sample = extract_csv(file(params.input, checkIfExists: true))
 
-if (params.wes) {
+if (params.wes && !params.step == 'annotate') {
```

Comment:
Ah, so you fixed it already 👍

Reply:
yeah since it was a one liner at roughly fit into the theme: fix intervals :D